scripts/apps/relations/directives/RelatedItemsDirective.ts

```diff
@@ -45,13 +47,31 @@ export function RelatedItemsDirective(authoringWorkspace, relationsService) {
                     event.preventDefault();
                     event.stopPropagation();
 
+                    const relatedItemsForCurrentField = Object.keys(scope.item.associations || {})
+                        .filter((key) =>  key.startsWith(scope.field._id) && scope.item.associations[key] != null)
+                        .map((key) => scope.item.associations[key]);
```

Comment:
Looks like map call is not needed, below you need only the list of keys.

Reply:
I need to retrieve `_id` at line 73

Reply:
yes, true